test/server/github.test.js

```diff
@@ -1,12 +1,13 @@
 import sinon from 'sinon'
 import { expect } from 'chai'
 import { GithubService } from '../../server/service/GithubService'
+import {toGenericComment} from '../../common/util'
 
 describe('The Github service', () => {
   let github
   const MOCK_COMMENTS = [
-    {created_at: '2016-07-11T15:00:00Z'},
-    {created_at: '2016-07-11T17:00:00Z'}
+    {created_at: '2016-07-11T15:00:00Z', user: {login: 'foo'}},
+    {created_at: '2016-07-11T17:00:00Z', user: {login: 'bar'}}
```

Comment:
In this particular case the `user` is not important as we only test if the date filter works and check the length of the returned array. That's why I deleted it.

But yes... Comments from Github have `user` as an object. We convert them now to a generic object where `user` is a string (to handle them consistently with comments from the database).

Somebody should build `clojure.spec` in JS.